tests/benchmarks/i2c_endless/boards/nrf54l15dk_nrf54l15_cpuapp.overlay

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
Support for PDK can be remove for this tests - no need to keep pdk files, only dk can stay.

Reply:
done